src/ontology/edits/oeo-physical.omn

```diff
@@ -1789,7 +1854,7 @@ Class: ParticulateMatter
         <http://purl.obolibrary.org/obo/IAO_0000119> "https://en.wikipedia.org/w/index.php?title=Particulates&oldid=906795232"@en
     
     SubClassOf: 
-        AirPollutant
+        AirPollution
```

Comment:
```suggestion
        PortionOfMatter
```
AirPollution is a process/emission, ParticulateMatter is a portion of matter which is emitted.